Source/WebKit/Shared/Cocoa/WKProcessExtension.mm

```diff
@@ -28,28 +28,22 @@
 
 #import <wtf/RetainPtr.h>
 
-static RetainPtr<WKProcessExtension>& sharedInstance()
+namespace WebKit {
+
+static RetainPtr<NSObject>& sharedProcessExtension()
 {
-    static NeverDestroyed<RetainPtr<WKProcessExtension>> instance;
+    static NeverDestroyed<RetainPtr<NSObject>> instance;
     return instance.get();
 }
 
-@implementation WKGrant
-@end
-
-@implementation WKProcessExtension
-+ (WKProcessExtension*) sharedInstance
+void setProcessExtension(NSObject* instance)
 {
-    return sharedInstance().get();
```

Comment:
I think you should RELEASE_ASSERT here that the `instance` responds to the selectors we expect.

Reply:
That is a good point. I added the RELEASE_ASSERT at the call site for performance reasons. 

Thanks for reviewing!